lib/apiservers/engine/backends/container.go

```diff
@@ -1699,3 +1391,42 @@ func ContainerSignal(containerID string, sig uint64) error {
 
 	return nil
 }
+
+//----------------------------------
+// ContainerLogs() utility functions
+//----------------------------------
+
+// validateContainerLogsConfig() validates and extracts options for logging from the
+// backend.ContainerLogsConfig object we're given.
+//
+// returns:
+//	tail lines, since (in unix time), follow
+func (c *Container) validateContainerLogsConfig(vc *viccontainer.VicContainer, config *backend.ContainerLogsConfig) (int64, int64, bool, error) {
+	if !(config.ShowStdout || config.ShowStderr) {
+		return 0, 0, false, fmt.Errorf("You must choose at least one stream")
+	}
+
+	// Validate tail lines and since params for future purposes, but VIC
+	// does not yet support them.
+	tailLines, err := strconv.ParseInt(config.Tail, 10, 64)
+	if err != nil {
+		tailLines = -1
+	}
+
+	var since time.Time
+	if config.Since != "" {
+		s, n, err := timetypes.ParseTimestamps(config.Since, 0)
+		if err != nil {
+			return 0, 0, false, err
+		}
+		since = time.Unix(s, n)
+	}
+
+	// Check if container is running
+	running, _ := c.containerProxy.ContainerRunning(vc)
```

Comment:
Why have this as a check and not an exception on the portlayer side?  So if you request the logs of a stopped container then throw an error instead of checking before hand?